test/browser/httpismSpec.js

```diff
@@ -219,4 +219,17 @@ describe('httpism', function () {
       });
     });
   });
+
+  describe('jsonReviver', function() {
+    it('uses the JSON reviver to decode JSON values', function() {
+      return httpism.get('/', {
```

Comment:
with this back-end you can `POST` an object and it will be returned in the `body` field of the response.